Frontend/src/Components/Battery/Battery.js

```diff
@@ -0,0 +1,143 @@
+                        {props.data?.fan_speed[0] ?? -1}
+                    </Text>
+
+                    {/* Hold the RangeBar within a vertical Flex to vertically center it */}
+                    <Flex direction='column'>
+                        <Spacer />
+                        <RangeBar 
+                            w={props.rangeBarW ?? '3em'} 
+                            h='1em'
+                            borderRadius='0px' 
+
+                            val={(props.data?.fan_speed[0] ?? -1.0)}
+                            min={fanSpeedMin}
+                            max={CONSTANTS.fan_speed.MAX}
+                        />
+                        <Spacer />
+                    </Flex>
+                </Flex>
+            </Center>
+            <RangeRow
+                dataTitle="Pack Current"
+                dataValue={props.data?.pack_current[0] ?? -1.0}
+                dataConstant={CONSTANTS.pack_current}
+
+                bg={headerBg}
+            />
+            <RangeRow
+                dataTitle="Pack Voltage"
+                dataValue={props.data?.pack_voltage[0] ?? -1.0}
+                dataConstant={CONSTANTS.pack_voltage}
+            />
+            <RangeRow
+                dataTitle="Pack Power Out"
+                dataValue={props.data?.pack_power[0] ?? -1.0}
+                dataConstant={CONSTANTS.pack_power}
+
+                bg={headerBg}
+            />
+            <RangeRow
+                dataTitle="Pack SoC"
+                dataValue={props.data?.soc[0] ?? -1.0}
+                dataConstant={CONSTANTS.soc}
+            />
+            <RangeRow
+                dataTitle="Pack Resistance"
+                dataValue={props.data?.pack_resistance[0] ?? -1.0}
+                dataConstant={CONSTANTS.pack_resistance}
+
+                bg={headerBg}
+            />
+            <RangeRow
+                dataTitle="Adapt. Total Capacity"
+                dataValue={props.data?.adaptive_total_capacity[0] ?? -1.0}
+                dataConstant={CONSTANTS.adaptive_total_capacity}
+            />
+            <Center
+                borderTopWidth={1}
+                borderBottomWidth={1}
+                borderRightWidth={1}
+                borderColor={borderColor}
+                bg={headerBg}
+                lineHeight='1.2em'
+                height='100%'
+            >
+                <Flex w='95%'>
+                    <Text fontSize='sm' lineHeight='2em'>Charge Enable:</Text>
+                    <Spacer />
+                    <Center
+                        border='2px'
+                        borderRadius='0px'
+                        h='75%'
+                        background={chargeEnableColor}
+                        marginTop='1%'
+                        marginBottom='1%'
+                    >
+                        <Text fontSize='xs' margin='0.7px 2px 0.7px'><strong>{chargeEnableText}</strong></Text>
```

Comment:
```suggestion
                        <Text
                            fontSize='xs'
                            fontWeight='bold'
                            lineHeight={1}
                            color={chargeEnableTextColor}
                        >
                            {chargeEnableText}
                        </Text>
```
Same as previous comment